attic/atom_mapping/mcs_atom_compare.py

```diff
@@ -18,3 +18,29 @@ def compare(self, p, mol1, atom1, mol2, atom2):
         x_i = mol1.GetConformer(0).GetPositions()[atom1]
         x_j = mol2.GetConformer(0).GetPositions()[atom2]
         return bool(np.linalg.norm(x_i - x_j) <= self.threshold)  # must convert from np.bool_ to Python bool!
+
+
+
+class CompareDistNonterminal(rdFMCS.MCSAtomCompare):
```

Comment:
This was only used in one test of `setup_relative_restraints_using_smarts` that could just as well use any smarts string -- modified this test to generate smarts using MCS with `rdFMCS.AtomCompare.CompareAnyHeavyAtom` instead of MCS with this custom atom comparison, and moved this custom atom comparison to attic for future reference

Reply:
Further simplified the test to use a smarts string rather than go through (conformer-aware or graph-only) MCS at all in https://github.com/proteneer/timemachine/pull/812/commits/eb541de292986245b175198e228009c89781d0d9